uiowa/uiowa
# [IWP] Rabbit hole taxonomy term permissions fix

Closes #8014 

# How to test
- `ddev blt ds --site=iwp.uiowa.edu`
- Masquerade as editor, publisher, and webmaster, and check that you can now actually view taxonomy term pages like https://iwp.uiowa.ddev.site/countries/afghanistan
